variables.tf

```diff
@@ -40,11 +50,14 @@ variable "definition" {
           - retentionDuration (optional) (default: 604800)
           - retainAcked (optional) (default: false)
           - expirationTtl (optional) (default: 2678400)
-  EOF
+  
+EOF
+
```

Comment:
why this change? Indent seems to be weird and empty lines unnecessary, same few lines above, of course, if this is caused by tf fmt it is ok, but it doesn't look like...

Reply:
I think vscode added this, I will see if I can fix it.